test/object-store/benchmarks/object.cpp

```diff
@@ -304,7 +305,7 @@ TEST_CASE("Benchmark object", "[benchmark][object]") {
         std::mt19937 rng(dev());
         std::uniform_int_distribution<std::mt19937::result_type> dist(500, 5000);
         int64_t benchmark_pk = dist(rng);
-        for (size_t i = 0; i < 1000; ++i) {
+        for (int64_t i = 0; i < 1000; ++i) {
```

Comment:
int64_t + size_t = size_t, so `size_t` here resulted in `benchmark_pk + i` being `size_t` and thus the `any_cast<int64_t>()` when reading the value threw.